### PR TITLE
Added support for Alpine.js

### PR DESCRIPTION
**Pull request by B-R-P**
Added alpine attributes similar to [htmx.d.ts](https://github.com/kitajs/html/blob/master/htmx.d.ts).

### CODE REVIEW

**Comment by arthurfiorette** [comment]
Hey @B-R-P thanks for your work! Would you mind documenting it inside the README.md file?

File: alpine.d.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+     *
+     * @see https://alpinejs.dev/directives/transition
+     */
+    'x-transition'?: string;
+      /**
+     * Two-way data binding for form elements.
+     *
+     * @see https://alpinejs.dev/directives/model
+     */
+    'x-model'?: string;
```
Comment (arthurfiorette) [inline alpine.d.ts:96]:
```suggestion
```

remove duplicated field

File: alpine.d.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
+declare namespace JSX {
```
Comment (arthurfiorette) [inline alpine.d.ts:1]:
```suggestion
// This file is a result from https://alpinejs.dev/
// Missing something? Please submit a issue report or a PR:
// https://github.com/kitajs/html

declare namespace JSX {
```

**Review by arthurfiorette — CHANGES_REQUESTED** [review]

File: alpine.d.ts (hunk 1 above)
Comment (B-R-P) [reply alpine.d.ts]: Removed in 7d56ab7

**Review by arthurfiorette — APPROVED** [review]

**Comment by B-R-P** [comment]
> Would you mind documenting it inside the README.md file?

Done in f3f2e03

**Comment by arthurfiorette** [comment]
thanks!

**Pull request merged**